guide/faq.md

```diff
@@ -24,7 +24,7 @@ Learn more by reading the [TELUS Design Platform roadmap](roadmap.md).
 
 - Chrome
 - Firefox
-- Safari: 10
+- Safari: 10 and higher
```

Comment:
Maybe we should just change this to Safari? Chrome, Firefox, Edge and Opera don't have version numbers next to their names. (IE 11 is an exception since it's a legacy browser and will never go past 11)